ui/DesignSystem/Component/ProjectList.svelte

```diff
@@ -8,11 +8,14 @@
 
   export let projects: Project[];
   export let userUrn: string;
+  export let orgAddress: string | undefined = undefined;
```

Comment:
If I understand the domain model correctly, this property is set if the project has an anchor. Instead of passing along the org address everywhere separately we should make this information part of the anchor data.

Reply:
fd795683630d4f44b822839e02b0d39360579df4